lldb/source/Utility/Stream.cpp

```diff
@@ -70,6 +72,31 @@ size_t Stream::PutCString(llvm::StringRef str) {
   return bytes_written;
 }
 
+void Stream::PutCStringColorHighlighted(llvm::StringRef text,
+                                        const char *pattern) {
+  if (!pattern) {
+    PutCString(text);
+    return;
+  }
+
+  // If pattern is not nullptr, we should use color
```

Comment:
Nit: seems more informative to have the inverse of this comment at the start of the function where we have an early return for the trivial case of not having a patter to highlight.

Reply:
Nice. Changed this.